conda/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set data= load_setup_py_data() %}
+package:
+  name: ts-interlockmonitor
+  version: {{ data.get('version') }}
+
+source:
+  path: ../
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install --no-deps --ignore-installed .
+  script_env:
+    - PATH
+    - PYTHONPATH
+    - LD_LIBRARY_PATH
+    - OSPL_HOME
+    - PYTHON_BUILD_VERSION
+    - PYTHON_BUILD_LOCATION
+    - LSST_DDS_DOMAIN
```

Comment:
How much of this do you actually need? This package doesn't use DDS so no OSPL_HOME and LSST_DDS_DOMAIN. Please ditch everything you can.

Reply:
The `script_env` session can be completely removed...